packages/entity-browser/src/routes/detail/components/DetailForm/DetailForm.js

```diff
@@ -40,6 +42,7 @@ export class DetailForm extends React.Component {
     return (
       <Field
         key={key}
+        readOnlyForm={this.isReadOnlyForm}
```

Comment:
A single field also can be readonly if the whole form is not readonly. In my opinion it should be something like {this.isReadOnlyForm || formDefinitionField.isReadOnly}